src/main/kotlin/com/amazon/elasticsearch/replication/task/index/IndexReplicationTask.kt

```diff
@@ -393,6 +397,17 @@ class IndexReplicationTask(id: Long, type: String, action: String, description:
         }
     }
 
+
```

Comment:
Please confirm if you have merge the subsequent bug fixes as well as part of this PR

Reply:
Yes, Only 2.x versions bug fix was done separately, for all 1.x and ES7.10 versions null pointer was handled in backport commit only